### DISCUSSION TITLE
Hacktober fest label

### DISCUSSION THREAD

**Discussion by prajna-spec** [BVP-ISTE/HacktoberFest2021#381] [General]
Hello!
Does the hacktober fest label indicate that the pull request is accepted. My pull request has a hacktober fest label but it doesn't show up as accepted on the website

**Comment by SohilK03** [comment]
It takes time for PR's to get recognized by the Hacktober site...check after 2-3 days ...you'll see the PR's mentioned.

till then keep contributing 😇

**Comment by prajna-spec** [comment]
Hello,
Thanks for the quick response. Its say matures in 14 days . Is it a review period or something like that?
